pyrk/version.py

```diff
@@ -31,7 +31,7 @@
 #
 #   include RELEASE-VERSION
 
-__all__ = ("get_git_version")
+__all__ = ["get_git_version"]
```

Comment:
It's just a list rather than a tuple. it clearly only has one element, but it compiles the version number hash better in sphinx this way.